test/read-files-test.js

```diff
@@ -0,0 +1,120 @@
+'use-strict';
+
+const fs = require('fs');
+const expect = require('chai').expect;
+const index = require('../lib/read-files.js');
+
+// let stuff = fs.readFile(`${__dirname}/../assets/stuff.txt`, (err, data) => {
```

Comment:
Sorry this was such a drag. Just an FYI to take out commented code in the future. :)